pages/linux/xdg-mime.md

```diff
@@ -0,0 +1,20 @@
+# xdg-mime
+
+> Query and manage MIME types according to the XDG standard.
+> More information: <https://portland.freedesktop.org/doc/xdg-mime.html>.
+
+- Find the MIME type of a file:
+
+`xdg-mime query filetype {{path/to/file}}`
+
+- Find the current default application for opening PNG images:
+
+`xdg-mime query default {{image/png}}`
+
+- Find the current default application for opening a file:
```

Comment:
```suggestion
- Display the default application for opening a specific file:
```

Reply:
i've commited it. it isn't really for a specific file, but for the file/MIME type of that file. e.g. if it's a file.png then it returns its MIME type image/png and query the default app for that. i originally wasn't more accurate in the phrasing (eg "Find default app for opening files with the same mime type as a file") because i thought it would add unnecessary wording.

Reply:
But in this example you're using a specific file, even if the file name is just `filename` without extension, the MIME type is correct. Maybe `same mime type as a file` is more correct, but is too long.